WordPress/src/main/res/values/strings.xml

```diff
@@ -140,14 +140,20 @@
     <string name="access_media_permission_required">Permissions required in order to access media</string>
     <string name="add_media_permission_required">Permissions required in order to add media</string>
     <string name="media_fetching">Fetching media…</string>
-    <string name="upload_queued">Queued</string>
     <string name="media_file_type">File type: %s</string>
     <string name="media_file_name">File name: %s</string>
     <string name="media_uploaded_on">Uploaded on: %s</string>
     <string name="media_dimensions">Dimensions: %s</string>
     <string name="media_upload_error">Media upload error occurred</string>
     <string name="media_generic_error">Media error occurred</string>
 
+    <string name="media_upload_state_queued">QUEUED</string>
```

Comment:
So, I was testing this with lowercase actually, but it was converted to all caps (I don't know where and how). I was testing on Turkish and the `Deleting` text became `DELETİNG`, notice the `İ`, so that's why I went with this. I'll check if `android:textAllCaps` works as expected.